support/unix/build.sh

```diff
@@ -4,19 +4,16 @@ usage() {
   cat <<'END_USAGE'
 build.sh: generate notion's generic unix installation script
 
-usage: build.sh <notion> <launchbin> <launchscript>
-  <notion>        path to the binary `notion` executable
-  <node>          path to the binary `node` executable
-  <launchbin>     path to the binary `launchbin` executable
-  <launchscript>  path to the binary `launchscript` executable
+usage: build.sh [target]
+  [target]   build artifacts to use ('release' or 'debug', defaults to 'release')
 
 The output file is saved as ./install.sh.
 END_USAGE
 }
 
-if [ "$#" -ne 4 ]; then
-  usage >&2
-  exit 1
+target_dir='release'
+if [ -n "$1" ]; then
```

Comment:
Can you also add a test that if `$#` is not 0 or 1, and if `$1` is neither `release` nor `debug` nor empty, that it should call `usage()` and `exit 1`?

Reply:
Good call I'll fix that